packages/firestore/rollup.config.js

```diff
@@ -59,6 +59,35 @@ const browserPlugins = [
   terser(util.manglePrivatePropertiesOptions)
 ];
 
+// TODO - update the implementation to match all content in the declare module block.
+function declareModuleReplacePlugin() {
```

Comment:
All our custom rollup plugins exist in `scripts/build`. Can you move this plugin to a new file in that directory?

Reply:
moved to `scripts/build`. Thanks.